src/main.rs

```diff
@@ -630,9 +681,28 @@ compatibility with the original codemod.",
                 .value_name("EXTENSION")
                 .multiple(true)
                 .require_delimiter(true)
-            // TODO: support Unix pattern-matching of extensions?
+                .conflicts_with_all(&["glob", "iglob"])
+                // TODO: support Unix pattern-matching of extensions?
                 .help("A comma-delimited list of file extensions to process."),
         )
+        .arg(
+            Arg::with_name("glob")
+            .short("g")
+            .long("glob")
+            .value_name("GLOB")
+            .multiple(true)
+            .require_delimiter(true)
+            .conflicts_with("iglob")
+            .help("A comma-delimited list of globs to process.")
```

Comment:
Can we do ripgrep-style multiple instead? (no delimiter, pass multiple times for multiple globs) I know this way is consistent with how codemod does it, but I think in this case codemod's behavior is a bit silly, so there's no need to carry it forward into the future. If you're coming from codemod, you get silly `--extensions` if you know about it, but if not you can have nice globs.

Reply:
Sure, I can adjust this no problem.